portal/src/it/scala/cromwell/pipeline/controller/ProjectControllerItTest.scala

```diff
@@ -29,56 +29,73 @@ class ProjectControllerItTest
   override protected def beforeAll(): Unit = {
     super.beforeAll()
     components.datastorageModule.pipelineDatabaseEngine.updateSchema()
+    userRepository.addUser(stranger).flatMap { _ =>
+      userRepository.addUser(dummyUser).flatMap { _ =>
+        projectRepository.addProject(dummyProject).map(_ => ())
+      }
+    }
```

Comment:
Nice! But you still need to explicitly await until it completes. Otherwise it is possible that some tests will start before these actions are complete, which will result in random failures.

Reply:
Ok, it was wrapped in Await.result and I hope it is normal.